src/bci_build/package/ruby.py

```diff
@@ -58,7 +58,18 @@ def _get_ruby_kwargs(ruby_version: Literal["2.5", "3.4"], os_version: OsVersion)
         },
         # as we only ship one ruby version, we want to make sure that binaries belonging
         # to our gems get installed as `bin` and not as `bin.ruby$ruby_version`
-        "config_sh_script": "sed -i 's/--format-executable/--no-format-executable/' /etc/gemrc",
+        "config_sh_script": (
+            (
+                """update-alternatives --set rake /usr/bin/rake.ruby.ruby3.4; \\
+    update-alternatives --set rdoc /usr/bin/rdoc.ruby.ruby3.4; \\
+    update-alternatives --set ri /usr/bin/ri.ruby.ruby3.4; \\
+    rm -v /usr/bin/ruby && ln -s ruby.ruby3.4 /usr/bin/ruby; \\
```

Comment:
I thought we're not supposed to mess in `/usr/bin/`?

Reply:
Right, I intended to file a bugreport about the underlying issue. The rm -v is to make sure it fails when the issue is fixed.

But I can change this to /usr/local and add a test case in BCI-tests instead.